generic/ras.py

```diff
@@ -40,6 +40,14 @@ def setUp(self):
         architecture = os.uname()[4]
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
+        if 'PowerNV' in process.system_output('grep ^platform /proc/cpuinfo'):
```

Comment:
It makes sense. Anyway, we also have an `avocado.utils.cpu`. If I had to choose a way to handle this `/proc/cpuinfo` search, `process.system_output('grep ...')` would be my last option. But this, of course, is just a suggestion. Your code, your call :)